UseCases/SupplyChain/README.md

```diff
@@ -0,0 +1,41 @@
+# Broader Supply Chain Value Propositions and Use Cases
+To fully appreciate the value of the OSIM use cases 
+requires understanding of the broader supply chain uses
+in which OSIM plays a role.
+For example to see the value of information modeling to 
+software bill of materials (SBOM), 
+you first have to undershand the value of SBOMs.
+
+The use cases in this directory are broader than OSIM 
+and show the value of supply chain transparence in general.
+
+The purpose of these use cases is to demonstrate the benefits of 
+Software Bills of Materials (SBOMs) to software Producers and Consumers. 
```

Comment:
Maybe normal English spelling would ease and still serve the reader?

> software bills of materials (SBOMs) to software producers and consumers.

Reply:
added to PR76 but done slightly differently since the particular explanation of acronym was redundant.